app/src/main/java/ju/video/player/view/playlist/filterspanel/DatePicker.java

```diff
@@ -1,99 +1,145 @@
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprieraty information of Jalasoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * Licence agreement you entered into with Jalasoft
+ */
 package ju.video.player.view.playlist.filterspanel;
  
 import java.awt.GridLayout;
 import java.awt.Dimension;
-import java.awt.Color;
 import java.awt.BorderLayout;
-
-import java.awt.event.ActionListener;
 import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import java.util.Calendar;
+import java.text.SimpleDateFormat;
 
 import javax.swing.JLabel;
 import javax.swing.JDialog;
 import javax.swing.JButton;
 import javax.swing.JPanel;
 
+import ju.video.player.view.commons.Button;
+import ju.video.player.view.commons.UIColor;
+import ju.video.player.view.commons.Fonts;
+
+/**
+ * It is the Date Piscker Dialog
+ *
+ * @author Adriana Olivera
+ * @version 1.0
+ */
+
 public class DatePicker {
-	int month = java.util.Calendar.getInstance().get(java.util.Calendar.MONTH);
-	int year = java.util.Calendar.getInstance().get(java.util.Calendar.YEAR);;
-	JLabel l = new JLabel("", JLabel.CENTER);
-	String day = "";
-	JDialog d;
-	JButton[] button = new JButton[49];
+	private static final int WEEKDAYS = 7;
```

Comment:
Maybe it's just my impression, but I think it's tabbed more than the necessary 4 spaces. If it is a visualization problem of mine, please ignore this comment.

Reply:
please, in my computer I see all normal, I don't kwon if it is a problem of my configuration or of the gift, please

Reply:
Must be a git bug